### PR TITLE
enhancement: Improve README.md

### PR DESCRIPTION
**Pull request by bishal7679**
# Fixes ⇾ #55

- Added this README.md for initial start
- Proposed the harbor cli logo to make it more attractive
- Needs a lot of improvements with further progress ...

//cc @Vad1mo @amands98
PTAL!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,74 @@
...
-  get         get project, registry, etc.
-  help        Help about any command
-  list        list project, registry, etc.
-  login       Log in to Harbor registry
-  update      update registry, etc.
+# Demo Screenshot
 
-Flags:
-  -h, --help   help for harbor
+![demo-1](https://github.com/goharbor/harbor-cli/assets/70086051/63b5f6b7-319b-4c05-968a-59489f7fdd35)
```
Comment (amands98) [inline README.md:20]: We don't need a screenshot; we can use Markdown to display available commands. Please check out the latest changes, which include users and repository commands as well.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,74 @@
...
+tar -xzf harbor_0.0.1_linux_amd64.tar.gz
+cd harbor_0.0.1_linux_amd64
+sudo mv harbor /usr/local/bin/
+
+## MacOS
+tar -xzf harbor_0.0.1_darwin_amd64.tar.gz
+cd harbor_0.0.1_darwin_amd64
+sudo mv harbor /usr/local/bin/
 ```
+## Windows
```
Comment (amands98) [inline README.md:54]: We're on Winget now. Can you add the Winget installation command?

Comment (bishal7679) [reply README.md:54]: ok!

Comment (bishal7679) [reply README.md:54]: @amands98 Could you pls provide me the actual command as I dont have any windows environment to test it out

Comment (amands98) [reply README.md:54]: `winget install harbor`

**Comment by Vad1mo** [comment]
@bishal7679 bravo!

**Pull request merged**